tasks/main.yml

```diff
@@ -159,3 +159,23 @@
   tags:
     - "amsrc-acceptancetests"
   when: "archivematica_src_install_acceptance_tests|bool"
+
+#
+# Configure SS
+#
+
+- include: "ss-configure.yml"
```

Comment:
As the files have only two tasks each, what about consolidating them into  a single one?